firmware/Makefile

```diff
@@ -387,5 +400,11 @@ ULIBS = -lm --specs=nano.specs
 
 include $(RULESFILE)
 
+openblt:
+	make -C $(PROJECT_DIR)/config/boards/$(PROJECT_BOARD)/openblt/
```

Comment:
i am in favor of bootloader pre-combilation mainly because I compile rusEFI too often. on the bright side I would not be affected as long as we do not enable bootloader into default 407 Frankenso bundle

Reply:
Build on request and keep prebuild images in git? Same strategy as for autogenerated.

Reply:
I do not insist on prebuild images in git, that's something we can decide on later. just a note that compilation time matters - specifically default build compilation time

Reply:
I am strongly against prebuilt images in git - we don't have to compile this for every board, only ones that actually need it.

Reply:
@mck1117 but the cheapest maintenance is to have it on ALL builds, and having in ALL builds would mean longer compile

Reply:
This change, as it sits, only compiles it when we need it.

Reply:
For the next 60 days we absolutely do NOT need prebuilt images. Let's move forward with everything else for now.